src/components/learner-credit-management/data/utils.js

```diff
@@ -12,10 +12,11 @@ import {
 export const transformOfferSummary = (offerSummary) => {
   if (!offerSummary) { return null; }
 
-  const totalFunds = parseFloat(offerSummary.maxDiscount);
-  const redeemedFunds = parseFloat(offerSummary.amountOfOfferSpent);
-  const remainingFunds = parseFloat(offerSummary.remainingBalance);
-  const percentUtilized = parseFloat(offerSummary.percentOfOfferSpent);
+  // Guard against potential bad data
```

Comment:
nit/curious: we could keep `parseFloat` juuuust in case the API decides to change one day to strings, it won't break the app.